babel.config.js

```diff
@@ -4,4 +4,5 @@ module.exports = {
         "@babel/preset-react",
         "@babel/preset-typescript",
     ],
+    plugins: ["@babel/plugin-proposal-nullish-coalescing-operator"],
```

Comment:
`@babel/preset-env` now handles this. Can you bump the babel packages?

Reply:
Done!